packages/widget-message-meet/README-USAGE.md

```diff
@@ -1,28 +1,109 @@
-# How to use the Message Meet Widget
+# Spark Message and Meet Widget
 
-Our widget is built using React (https://github.com/facebook/react), Redux (https://github.com/reactjs/redux), and the Spark JS SDK (https://github.com/ciscospark/spark-js-sdk).
+The Spark Message and Meet widget allows developers to quickly and easily incorporate messaging through Cisco Spark into an application. This widget handles the heavy lifting of coordinating between your application and the Spark APIs, and provides a complete Spark UI and messaging experience without having to build all front end UI yourself.
```

Comment:
I'm not sure I like the wording after the last "and". It's kinda intentionally *not* the complete experience.